easybuild/easyblocks/generic/tarball.py

```diff
@@ -43,6 +48,20 @@ class Tarball(EasyBlock):
     - will unpack binary and copy it to the install dir
     """
 
+    @staticmethod
+    def extra_options(extra_vars=None):
+        """Extra easyconfig parameters specific to Tarball."""
+        extra_vars = EasyBlock.extra_options(extra=extra_vars)
+        extra_vars.update({
+            'install_in_subdir': [False, "Extract tarball in a subdirectory with the name of the package", CUSTOM],
+            'preinstall_cmd': [None, "Command to execute before installation", CUSTOM],
+        })
```

Comment:
'dirs_exist_ok': [False, "Merge into existing folder instead of throwing Exception if installdir exists.", CUSTOM],

Reply:
I added a single option called `install_type` covering both `install_in_subdir` and `dirs_exist_ok` as they should be exclusive.